source/contribute/maintainer.rst

```diff
@@ -59,16 +67,167 @@ Optionally, add any additional contributor/maintainer forks, e.g.:
 
     $ git remote add romanb git://github.com/romanb/doctrine2.git
```

Comment:
This could be reworded to `<username>` for consistency with the change in `git clone ...`.